src/Compilers/VisualBasic/Test/Emit/Emit/NoPiaEmbedTypes.vb

```diff
@@ -4176,9 +4177,13 @@ BC35000: Requested operation is not available because the runtime library functi
                                                     Assert.Equal(ParameterAttributes.HasDefault, p.ParamFlags)
                                                     Assert.Equal(CObj(&H0000000C), p.ExplicitDefaultConstantValue.Value)
                                                     Assert.False(p.HasExplicitDefaultValue)
-                                                    Assert.Throws(GetType(InvalidOperationException), Sub()
-                                                                                                          Dim tmp = p.ExplicitDefaultValue
-                                                                                                      End Sub)
+                                                    If InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter.IsUnavailable Then
+                                                        Assert.Throws(GetType(InvalidOperationException), Sub()
```

Comment:
Situation is for when the default language version supports this feature, as will no longer throw.